examples/interaction/legends/legend_hide.py

```diff
@@ -1,3 +1,16 @@
+''' The legend_hide feature is use to hide corresponding lines on a plot.
+This examples demonstrates an interactive line chart for stock prices over
+time which allows the user to click on the legend entries to hide or show
+the corresponding lines.
+
+.. bokeh-example-metadata::
+    :sampledata: bokeh.sampledata.stocks
+    :apis: bokeh.plotting.figure.line, bokeh.palettes.Spectral4
+    :refs: :ref:`ug_basic_annotations_legends`
+    :keywords: line chart, time series, legend_hide, interactive, legend
```

Comment:
I'd suggest paring down and making a bit more general
```suggestion
    :keywords: line, time series, legend
```